Watchman.Discord/Areas/Responses/Services/ResponsesMessageService.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Devscord.DiscordFramework.Commons.Extensions;
+using Devscord.DiscordFramework.Framework.Commands.Responses;
+using Devscord.DiscordFramework.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Watchman.Discord.Areas.Responses.Services
+{
+    public class ResponsesMessageService
+    {
+        private const string DESCRIPTION = "dokumentacja:\nhttps://watchman.readthedocs.io/pl/latest/135-services-in-framework/";
+        private ResponsesGetterService _responsesDatabase;
+        private EmbedMessageSplittingService _embedMessageSplittingService;
+        private readonly ulong _serverId;
+
+        public ResponsesMessageService(ResponsesGetterService responsesDatabase, EmbedMessageSplittingService embedMessageSplittingService, ulong serverId)
```

Comment:
nie przyjmuj serverId, bo nie będzie dało się zbudować tej klasy autofaciem

Reply:
to jak mam pobrać id aktualnego serwera?

Reply:
jako argument dla danej metody, po prostu zamiast przekazywać konstruktorowi to przekazuj metodzie która będzie tego potrzebowała